apps/nextjs-website/src/_contents/webinars.ts

```diff
@@ -55,7 +55,7 @@ export const webinars: readonly Webinar[] = [
       `<p>Esploriamo il nuovo modello di comunicazione su app IO, che consente nuovi scenari d’uso e nuove esperienze utente grazie alle tecniche di remotizzazione.
        </p> \n` +
       `<br /> \n` +
-      `<img style="height: auto;max-width: 100%" src='/images/webinar-io-remote-content.png' /> \n` +
+      `<img style="height: auto;max-width: 820px" src='/images/webinar-io-remote-content.png' /> \n` +
```

Comment:
Without `100%` max-width on small viewport or device image overflow page view.
```suggestion
      `<img style="height: auto;width: 820px;max-width: 100%" src='/images/webinar-io-remote-content.png' /> \n` +
```

Reply:
I inverted width and max-width from your suggestion, the max-width should be set to `820px` or else on desktop viewports the image results too large